legacy/configadmin/src/test/resources/logging.properties

```diff
@@ -24,7 +24,7 @@
 #loggers=org.jboss.whatever,org.jboss.foo
 
 # Root logger level
-logger.level=DEBUG
+logger.level=OFF
```

Comment:
Should this really be off? Wouldn't it be better to have it on ERROR to help diagnose any problems

Reply:
yes, it should be off, if you need to diagnose it, you enable it right than, in 99.999% of the cases when the testsuite runs, which is on CI, noone will ever look at this.